tests/common.bash

```diff
@@ -743,7 +751,7 @@ function get_dep_from_yaml_db(){
 
         "${repo_root_dir}/ci/install_yq.sh" >&2
 
-        result=$("${GOPATH}/bin/yq" r -X "$versions_file" "$dependency")
+        result=$("${GOPATH}/bin/yq" "$dependency" "$versions_file")
```

Comment:
Do we need to prepend the `.` in front of dependency here, or have all callers of this been updated?

Reply:
I have updated the caller at line 770.